MAP: Outpost prison

MrCat15352 opened the pull request:
## Описание / Что этот PR делает
Заказ для Копо

## Причина создания ПР / Почему это хорошо для игры
Заказ для Копо

## Демонстрация изменений / Тестирование

## Список изменений

```yml
🆑
add: Что-то добавлено
del: Что-то удалено
balance: Что-то изменено в балансе
tweak: Что-то изменено по мелочи
fix: Что-то исправлено
sound: Добавлен/Изменен/Удален звук
image: Добавлен/Изменен/Удален спрайт
map: Изменения на карте
spellcheck: Исправлена опечатка/добавлен перевод
admin: Изменения в кнопках администрации
/🆑
```

MrCat15352 committed:
Эта карта пойдет под будущий ивент

MrCat15352 committed:
Эта карта пойдет под будущий ивент

MrCat15352 committed:
Эта карта пойдет Копо на ивент

MrCat15352 committed:
Эта карта пойдет Копо на ивент

MrCat15352 committed:
Попытка реализовать воду

MrCat15352 committed:
Попытка реализовать воду

MrCat15352 committed:
Merge branch 'outpost_prison_event' of https://github.com/MrCat15352/MrCat into outpost_prison_event

MrCat15352 committed:
Продолжаем мапить

MrCat15352 committed:
Продолжаем мапить

MrCat15352 committed:
в комменты воду

MrCat15352 committed:
в комменты воду

MrCat15352 committed:
Создаем отдельный модуль для ивента

MrCat15352 committed:
Создаем отдельный модуль для ивента

MrCat15352 committed:
Правим карту

MrCat15352 committed:
Правим карту

MrCat15352 committed:
Фиксы тюрьмы

MrCat15352 committed:
Фиксы тюрьмы

MrCat15352 committed:
Чето помапил

MrCat15352 committed:
Чето помапил

MrCat15352 committed:
Создал заделку на ерт шип сф

MrCat15352 committed:
Создал заделку на ерт шип сф

MrCat15352 committed:
Добавляем ивентовую турельку

MrCat15352 committed:
Добавляем ивентовую турельку

MrCat15352 committed:
Merge remote-tracking branch 'upstream/beta-dev' into outpost_prison_event

MrCat15352 committed:
Готово. Карта готова, спавнеры готовы

MrCat15352 committed:
Готово. Карта готова, спавнеры готовы

MrCat15352 committed:
Merge branch 'CeladonSS13:beta-dev' into outpost_prison_event

MrCat15352 committed:
FIX: Капюшон нельзя теперь вытащить из толстовок (#2370)

## Описание / Что этот PR делает
При попытке взаимодействовать с капюшоном из карманов курток, ничего не
будет происходить, а в чат будет писаться что это нельзя сделать, а
именно вытащить капюшон. Для этого есть спец кнопка

## Причина создания ПР / Почему это хорошо для игры
При попытках вытащить капюшон, заставляло игру циклично спавнить
ошибочный спрайт. Над игроком появлялось слово - YOU ERROR


## Список изменений

```yml
🆑
add: проверка на съёмность капюшона от курток
/🆑
```

Mirag1993 committed:
FIX: Fix ethereal medecine (#2381)

## Описание / Что этот PR делает
Исправляем баг с отображением крови эльзуосов (жидкого электричества ) в
медецинских сканерах .
## Причина создания ПР / Почему это хорошо для игры
Эльзуосы вызывали баги отображения на мед сканерах из за нестандартного
вида крови
## Демонстрация изменений / Тестирование
<img width="767" height="584" alt="image"
src="https://github.com/user-attachments/assets/0ff52271-2efe-44b9-b5d8-1c3ee11aad2d"
/>
<img width="1571" height="607" alt="image"
src="https://github.com/user-attachments/assets/8f1e30f5-ff03-43d3-9617-1e2a624f06e5"
/>
<img width="907" height="212" alt="image"
src="https://github.com/user-attachments/assets/2223094e-80d8-4f5f-bc55-1cab203d2339"
/>

## Список изменений

```yml
🆑
fix: Взаимодействие эльзуосов с мед сканерами
/🆑
```

---------

Signed-off-by: Mirag1993 <145050764+Mirag1993@users.noreply.github.com>
Co-authored-by: KOCMOHABT <25280520+KOCMODECAHTHUK@users.noreply.github.com>

MrCat15352 committed:
ADD: Новый торговый автомат с одеждой - Сакура (#1872)

MrCat15352 committed:
FIX: Важный фикс спрайта (#2395)

MrCat15352 committed:
REFACTOR: Возвращаем часть тикет системы обратно в коркод (#2373)

## Описание / Что этот PR делает
Возвращаем на место метод fire сабсистемы тикетов, меняя на месте
коркод, без оверайда

## Причина создания ПР / Почему это хорошо для игры
Отделять частичку от тикета хоть и маловажнозначимую - не надо так
делать

## Список изменений

```yml
🆑
code: рефактор кода 
/🆑
```

Vairkharst commented:
> Опять же спрошу, это чисто под ТМ или на постоянку ты хочешь?

Это в ТМ , потом уже после ивента и фиксов на мейн сервер

MrCat15352 committed:
MAP FIX: Исправляем уголок магазина на зимнем аванпосту (#2403)

## Описание / Что этот PR делает
Исправил баг карты. Также закольцевал трубы атмоса и придал эстетическйи
завершенный вид этому участку

## Причина создания ПР / Почему это хорошо для игры
Баг

## Демонстрация изменений / Тестирование
<img width="621" height="397" alt="изображение"
src="https://github.com/user-attachments/assets/7ddc7ce1-34f4-466b-ab80-c737a9022915"
/>

## Список изменений

```yml
🆑
map: изменил небольшой участок между магазином и инженерным отделом на зимнем аванпосту
/🆑
```

MrCat15352 committed:
ADD: Переключение слоев для обуви и поясов (#2392)

## Описание / Что этот PR делает
Добавляет возможность на ctrl+клик по обуви или по поясам переключать их
между двумя слоями. Прятать под андер одежду или возвращать в обычный
режим отображения.

## Причина создания ПР / Почему это хорошо для игры
Заказ хэда спрайтеров и администратора 

## Демонстрация изменений / Тестирование
Обычный вид
<img width="134" height="141" alt="изображение"
src="https://github.com/user-attachments/assets/945d2520-5ca3-45bd-b7b2-503c9533922e"
/>

<img width="86" height="103" alt="изображение"
src="https://github.com/user-attachments/assets/cb3f6064-5aa9-4d5b-b10c-fec39a19b188"
/>

Переключенный вид
<img width="99" height="122" alt="изображение"
src="https://github.com/user-attachments/assets/665a5237-3312-4854-abf9-0872ddbbe8c2"
/>

<img width="131" height="133" alt="изображение"
src="https://github.com/user-attachments/assets/d3a9da59-232d-4ad3-98b2-a83893807d78"
/>

Пояс обычный
<img width="87" height="83" alt="image"
src="https://github.com/user-attachments/assets/1c9281ed-93e8-478c-aa75-333c46510170"
/>

Пояс под одеждой
<img width="96" height="87" alt="image"
src="https://github.com/user-attachments/assets/a0815b90-2783-4c87-83b2-67d573598f82"
/>


## Список изменений

```yml
🆑
add: механизм по переключению обуви
/🆑
```

MrCat15352 committed:
FIXES: Куртки, худи, а также шарфы и плащи (#2393)

## Описание / Что этот PR делает
ПР неотделимо делает две вещи. Первое чинит окончательно худи, куртки,
теперь внутри них не будет спавниться капюшон и вызывать ошибки
буквально
Второй момент, теперь плащи и шарфы можно как и раньше вешать на шею, и
активировать их

## Причина создания ПР / Почему это хорошо для игры
Заказ от хэда спрайтеров

## Демонстрация изменений / Тестирование

## Список изменений

```yml
🆑
add: возможность вешать с переключение шарфы и плащи на слот шеи
fix: древний баг с капюшоном почти полностью пофикшен (98%)
/🆑
```

---------

Signed-off-by: Voiko | Mr_DarkBladeS <48252701+MysticalFaceLesS@users.noreply.github.com>
Co-authored-by: Voiko | Mr_DarkBladeS <48252701+MysticalFaceLesS@users.noreply.github.com>

MrCat15352 committed:
FIXES: Автомат Сакура или то как я случайно снес половину спрайтов (#2394)

## Описание / Что этот PR делает
Собственно, думая что атласы не нужны со спрайтами - я снес их. Этот ПР
их возвращает и чинит спрайты

## Причина создания ПР / Почему это хорошо для игры
Быстрый хот фикс для одежды!

## Демонстрация изменений / Тестирование

## Список изменений

```yml
🆑
add: вернул потерянные атласы со спрайтами
/🆑
```

MrCat15352 committed:
UPDATE: Новое-старое меню лодаута (#2297)

## Описание / Что этот PR делает
Обновляем меню лодаута, чистим от всякого мусора, приводим в удобный вид
(хартстоун). Избавляемся от хаоса. Убираем таблицу и внедряем карточки
предметов. Чиним баги с ползунком, тееперь можно выбрать предмет и не
быть поднятым в начало списка, наконец-то пофикшено.

## Причина создания ПР / Почему это хорошо для игры
Меню лодаута было неудобным. Теперь оно необычное, чем-то на тг похожее,
но сделано не на базе ТГУИ

## Демонстрация изменений / Тестирование
<img width="1324" height="879" alt="image"
src="https://github.com/user-attachments/assets/b96a3334-60a4-4440-9c17-338a3cafb95a"
/>
<img width="1650" height="1071" alt="image"
src="https://github.com/user-attachments/assets/99f093a5-b462-488f-9044-d78e8ff89c8c"
/>
<img width="1646" height="1066" alt="image"
src="https://github.com/user-attachments/assets/1c352a75-5e11-4120-9612-9345202fe9b6"
/>
<img width="694" height="274" alt="image"
src="https://github.com/user-attachments/assets/aa4591be-774c-4957-bd7d-12f0d69d0531"
/>
<img width="811" height="325" alt="image"
src="https://github.com/user-attachments/assets/1cc8a195-7d09-4eec-ae23-39e0b26e6346"
/>

## Список изменений

```yml
🆑
tweak: меню лодаута
/🆑
```

---------

Signed-off-by: MrCat15352 <125268716+MrCat15352@users.noreply.github.com>

Kenfarka committed:
MAP: Talos tweak and some fixes (#2390)

## Описание / Что этот PR делает
Реворкает и чинит талос. Реворк от Ерринга
## Причина создания ПР / Почему это хорошо для игры
Чиним талос в некоторых моментах. Добавляем ему некую "уникальность".
Closed #2384
## Демонстрация изменений / Тестирование

## Список изменений

```yml
🆑

map:
Реворк в себя включает:
Мехбей со вторым мехом с лазерным карабином;
Эквип дополнительный для мехов в виде: их пилы, ремонтный дрон, трастеры (1 сет);
Оружейная потерпела изменение: 2 монгреля, добавлен 1 мастифф, добавлены 4 фонарика на оружие. В общем итоге теперь в оружейной 2 мастиффа, 2 пистолета;
Фикс прикола с хардсьютами;
Вендор с кофе всё ещё в бласт дурах;
Техтоннели были реворкнуты;
Мед был изменен, и вместо дюффеля там теперь хирургический набор;
Карго претерпело косметические изменения;
Поправлены немного доки для стыковки субов;
Добавлены свернутые флаги в зону отдыха;
Число турелей было увеличено до 16 (да) и они были изменены до средних;
Поправлены переключатели света, передвинуты АПЦ;
Передвинуты столы в столовой;
Интеркомы стали обычной частоты;
Теха на левом борту стали другими, менее запутанными через стенки, добавлен прикол;
Павер паки с гриндером для СЕ и инженерам 

/🆑
```

PumPeyMey committed:
MAP: Fixes shuttles (#2389)

## Описание / Что этот PR делает
Исправление ошибок у шаттлов.
## Причина создания ПР / Почему это хорошо для игры
Баги - плохо.
Closed #2396

## Демонстрация изменений / Тестирование

## Список изменений

```yml
🆑
fix: Замена неправильных плейтингов у Лалиша и Халифата, Фист оф фридом кнопки и турели более не дублируют чужие айди, Тирдроп турели правильно повернуты среди них более нет предателя, в меде и карго появились файер алярмы, в меде огнетушитель, Целестис заменил импланты раций на обычные рации, Биитл теперь имеет верное описание.
/🆑
```

---------

Co-authored-by: Kenfarka <namor4ikm@gmail.com>

MrCat15352 committed:
HOT-FIX: Дубликат кода переключения обуви (#2404)

## Описание / Что этот PR делает
Починил конфликт Войко

## Причина создания ПР / Почему это хорошо для игры

## Демонстрация изменений / Тестирование

## Список изменений

```yml
🆑
del: Дубликат кода
/🆑
```

MysticalFaceLesS committed:
Update CODEOWNERS to change maintainer names (#2405)

## Описание / Что этот PR делает

Изменяем кодовнера для карт.

## Причина создания ПР / Почему это хорошо для игры

Так как карты теперь проверять некому - кодовнером карт стану временно
я.

Signed-off-by: Voiko | Mr_DarkBladeS <48252701+MysticalFaceLesS@users.noreply.github.com>

MrCat15352 committed:
TOOLS: Скрипт для парсинга рантаймов (#2375)

## Описание / Что этот PR делает
Пока инструментарий для логов в разработке единый, нужна замена быстрая
для парсера логов рантаймов.

## Причина создания ПР / Почему это хорошо для игры
Нужны инструменты 

## Демонстрация изменений / Тестирование

## Список изменений

```yml
🆑
add: парсер логов рантайма
/🆑
```

MrCat15352 committed:
UPDATE: Ириска снова сияет (#2377)

## Описание / Что этот PR делает
Перерабатывает старый код Ириски делая её более живую, настырную, и
требовательную к еде

## Причина создания ПР / Почему это хорошо для игры
Старый код Ириски вызывал кучу проблем с игрой. Теперь он стал
оптимизированнее, а фишки Ириски вновь работают. Да-да, лапа что может
сломать позвоночник включена в апдейт

## Список изменений

```yml
🆑
code: Ириска оптимизирована
map: починены пути на картах
/🆑
```

Kenfarka committed:
MAP: выпуск шипов с тэгами из щитспавна (#2302)

## Описание / Что этот PR делает
выпускает оставшиеся шипы с тэгом из щитспавна. НО. Кроме некоторых,
которые нуждаются в реальном реворке.
## Причина создания ПР / Почему это хорошо для игры
Ради тестов. Настоящие тесты можно провести только на хайпоп сервере,
без этого никак
## Демонстрация изменений / Тестирование


```yml
🆑
map: выпускает оставшиеся шипы с тэгом из щитспавна: лагуна, немо, талос, караван, геко
/🆑
```

PumPeyMey committed:
MAP: RT Shuttle (#2380)

## Описание / Что этот PR делает
Заказ Копо
## Причина создания ПР / Почему это хорошо для игры
Заказ Копо
## Демонстрация изменений / Тестирование

## Список изменений

```yml
🆑
map: Шаттл для солов
/🆑
```

MrCat15352 committed:
-

MrCat15352 committed:
Merge branch 'outpost_prison_event' of https://github.com/MrCat15352/MrCat into outpost_prison_event
